ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+                this.breadcrumbsGroup.destroy();
+                this.breadcrumbsGroup = void 0 as any;
+                chart.getMargins();
+                chart.redraw();
+            } else {
+                const prevConector = breadcrumbsList[lastVisibleLevel + 1][3];
+
+                // Remove connector from the previous button.
+                prevConector && prevConector.destroy();
+                breadcrumbsList[lastVisibleLevel + 1].length = 3;
+
+                for (let i = lastVisibleLevel + 2; i < breadcrumbsList.length; i++) {
+                    const el = breadcrumbsList[i],
+                        button = el[2],
+                        conector = el[3];
+
+                    // Remove SVG elements fromt the DOM.
+                    button && button.destroy();
+                    conector && conector.destroy();
+                }
+                // Clear the breadcrums list array.
+                breadcrumbsList.length = lastVisibleLevel + 2;
+            }
+        } else {
+            const drilldownLevels = chart.drilldownLevels;
+
+            // Update button.
+            if (drilldownLevels && (!drilldownLevels.length || this.level === -1) && chart.drillUpButton) {
+                breadcrumbsList.length = 0;
+                chart.drillUpButton.destroy();
+                chart.drillUpButton = void 0;
+            } else if (chart.drillUpButton) {
+                chart.drillUpButton.attr({
+                    text: '◁ ' + breadcrumbsList[this.level][1].name
+                });
+            }
+            chart.getMargins();
+            chart.redraw();
+        }
+
+        if (breadcrumbsOptions && breadcrumbsOptions.position && breadcrumbsOptions.position.align !== 'left') {
+            breadcrumbs.alignGroup();
+        }
+    }
+
+    /**
+     * Create a group, then draw breadcrumbs together with the separators.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#draw
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public draw(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = chart.options.breadcrumbs,
+            lastBreadcrumbs = breadcrumbsList[breadcrumbsList.length - 2] &&
+                breadcrumbsList[breadcrumbsList.length - 2][2],
+            buttonPadding = breadcrumbsOptions && breadcrumbsOptions.buttonPadding;
+
+        // A main group for the breadcrumbs.
+        if (!this.breadcrumbsGroup && breadcrumbsOptions) {
+            this.breadcrumbsGroup = chart.renderer
+                .g('breadcrumbs-group')
+                .attr({
+                    zIndex: breadcrumbsOptions.zIndex
+                })
+                .addClass('highcharts-breadcrumbs')
+                .add();
+            chart.redraw();
```

Comment:
Why do we need to redraw the chart here? In most scenarios, we use `chart.redraw()` at the end (if needed).